docs/articles/deployment/toc.yml

```diff
@@ -7,4 +7,6 @@
 - name: Headless Service
   href: headless-service.md
 - name: Azure Cloud Service (Deprecated)
-  href: azure-cloud-service.md
\ No newline at end of file
+  href: azure-cloud-service.md
+- name: Troubleshooting Akka.Cluster
```

Comment:
Added to the deployment area of website for now - should I also add this in the TOC for Akka.Cluster itself?